lib/api/src/grpc/proto/points.proto

```diff
@@ -507,6 +507,15 @@ enum Fusion {
     RRF = 0; // Reciprocal Rank Fusion
 }
 
+/// Sample points from the collection
+///
+/// Available sampling methods:
+///
+/// * `random` - Random sampling
+enum Sample {
+    RANDOM = 0;
```

Comment:
Random thought here, is there a use case for exposing to precise underlying distribution used to sample ids?

```rust
enum Sample {
  RAND_UNIFORM = 0;
  RAND_NORMAL = 1;
  ...
} 
```

Reply:
I don't think that makes a lot of sense.

First of all, if I assume that 'normal' is the standard `rand` distribution, then it basically is the same as uniform.

Second, we pick random points directly from storage slices. Not from an ordered list of point IDs. And storage is modified all the time. I therefore think different distributions are meaningless. Correct me if I'm wrong though!

Reply:
> we pick random points directly from storage slices. Not from an ordered list of point IDs. And storage is modified all the time. I therefore think different distributions are meaningless. Correct me if I'm wrong though!

Basically this. But also a normal distribution assumes some mean to use as center, which also doesn't makes sense for me.

Reply:
If other distributions do not make sense here, any idea if this enum could actually receive additional variants in the future?

Reply:
@agourlay Now that I think about it, we could be adding **vector**-based distributions/approaches like:
- normal distribution (I think I missed that this is what you meant 🙈 )
- minimum similarity (aka diversity). Returning points which are furthest from all other returned points.

Reply:
I am fine for keeping it open until we figure something else out :+1: 

Thanks for thinking about it.